package.json

```diff
@@ -26,6 +26,7 @@
   "devDependencies": {
     "@ionic/cli-framework": "1.0.0-rc.5",
     "@types/dargs": "^5.1.0",
+    "@types/jasmine": "^2.8.6",
```

Comment:
How come these types are necessary for the base package?

Reply:
Without those types, VSCODE is showing errors when writing the tests in the starter folder (i.e. not the build)